pkgs/test_core/lib/src/runner/reporter/expanded.dart

```diff
@@ -62,21 +62,21 @@ class ExpandedReporter implements Reporter {
 
   /// The size of `_engine.passed` last time a progress notification was
   /// printed.
-  int _lastProgressPassed;
+  int? _lastProgressPassed;
```

Comment:
What's our policy with instances like this here. Should we make them non-nullable and default them to 0?

Reply:
done for all but one of these, I think that is reasonable